lib/tasks/edu.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+namespace :edu do
+  desc 'Given a confirmation number, print a spool file entry'
+  task :print, [:id] => [:environment] do |_t, args|
+    raise 'need to give an id. edu:print[{id}]' if args[:id].blank?
+    id = args[:id].gsub(/\D/, '').to_i
```

Comment:
Is there a particular thing like dashes that this is removing, or is it just a general catch-all?